queries/c/highlights.scm

```diff
@@ -112,10 +112,7 @@
 (number_literal) @number
 (char_literal) @character
 
-[
- (preproc_arg)
- (preproc_defined)
-]  @function.macro
+((preproc_arg)@function.macro (#set! "priority" 90))
```

Comment:
maybe a space after `(preproc_args)`? `((preproc_arg) @function.macro (#set! "priority" 90))`

Reply:
yep, sorry, I got sidetracked and forgot to finish that file!